src/NeoSharp.Core/Network/IServerContext.cs

```diff
@@ -9,5 +10,9 @@ public interface IServerContext
         /// Version
         /// </summary>
         VersionPayload Version { get; }
+
+        IEnumerable<EndPoint> ConfigurableEndpoints { get; }
+
+        NetworkAclConfig NetworkAclConfig { get; }
```

Comment:
This information is useful only for Server. Other consumers have no point to see this. Moreover, nobody cannot use it without INetworkAclLoader.

Reply:
The ServerContext is the DTO that support the Server class and where the server class can exchange informations with other classes.

Don't make any sense that that Server and ServerContext depend on the NetworkConfig. 
Because the NetworkConfig is a DTO, make all the sense to live in the ServerContext and the Server use it when need.

Reply:
I agree about NetworkConfig but what I meant is two new introduced properties are useful only for IServer and their exchange with other classes is questionable.